Makefile

```diff
@@ -1,6 +1,6 @@
 CFLAGS = -ggdb -w -Ilibtpm -std=c99
 
-LDLIBS=-Llibtpm -ltpm -lcrypto -loath -lqrencode
+LDLIBS=-Llibtpm -ltpm -lcrypto -loath -lqrencode -ltspi
```

Comment:
We don't want to link unsealtotp against libtspi, otherwise we have to drag a bunch of dependencies into the initramfs.